source/docs/tag-plugins.md

```diff
@@ -196,7 +196,39 @@ Inserts a link with `target="_blank"` attribute.
 Inserts code snippets in `source/downloads/code` folder. The folder location can be specified through the `code_dir` option in the config.
 
 ```
-{% include_code [title] [lang:language] path/to/file %}
+{% include_code [title] [lang:language] [from:line] [to:line] path/to/file %}
+```
+
+### Examples
+
+**Embed the whole content of test.js**
+
+```
+{% include_code lang:javascript test.js %}
+```
+
+**Embed line 3 only**
+
+```
+{% include_code lang:javascript from:3 to:3 test.js %}
+```
+
+**Embed line 5(inclusive) to 8(exclusive)**
```

Comment:
I don't think `inclusive` `exclusive` are necessary, "Embed line 5 to 8" is already clear enough. Applies to the rest and zh-cn.